docs/plugins/integrations/configuration.rst

```diff
@@ -0,0 +1,125 @@
+*************************
+Integration configuration
+*************************
+
+.. contents:: Table of contents
+
+The Integration Plugin provides interfaces to display and store configuration options that can be accessed through the ``\Mautic\PluginBundle\Entity\Integration`` object.
+
+Registering the Integration for configuration
+#############################################
+
+To tell the IntegrationsBundle that this Integration has configuration options, tag the Integration or support class with ``mautic.config_integration`` in the Plugin's ``app/config.php``.
+
+.. code-block:: php
+
+    <?php
+    return [
+        // ...
+        'services' => [
+            // ...
+            'integrations' => [
+                // ...
+                'helloworld.integration.configuration' => [
+                    'class' => \MauticPlugin\HelloWorldBundle\Integration\Support\ConfigSupport::class,
+                    'tags'  => [
+                        'mautic.config_integration',
+                    ],
+                ],
+                // ...
+            ],
+            // ...
+        ],
+        // ...
+    ];
+
+The ``ConfigSupport`` class must implement ``\Mautic\IntegrationsBundle\Integration\Interfaces\ConfigFormInterface``.
+
+.. code-block:: php
+
+    <?php
+    namespace MauticPlugin\HelloBundle\Integration\Support;
+
+    use MauticPlugin\HelloWorldBundle\Form\Type\ConfigAuthType;
+    use MauticPlugin\IntegrationsBundle\Integration\DefaultConfigFormTrait;
+    use MauticPlugin\IntegrationsBundle\Integration\Interfaces\ConfigFormInterface;
+    use MauticPlugin\IntegrationsBundle\Integration\Interfaces\ConfigFormAuthInterface;
+
+    class ConfigSupport implements ConfigFormInterface, ConfigFormAuthInterface
+    {
+        use DefaultConfigFormTrait;
+
+        public function getDisplayName(): string
+        {
+            return 'Hello World';
+        }
+
+        /**
+         * Return a custom Symfony form field type class that will be used on the Enabled/Auth tab.
+         * This should include things like API credentials, URLs, etc. All values from this form fields
+         * will be encrypted before being persisted.
+         *
+         * @link https://symfony.com/doc/2.8/form/create_custom_field_type.html#defining-the-field-type
+         *
+         * @return string
+         */
+        public function getAuthConfigFormName(): string
+        {
+            return ConfigAuthType::class;
+        }
```

Comment:
Is this still accurate? The provided link is for Symfony 2.8.

Reply:
New URL is https://symfony.com/doc/4.4/form/create_custom_field_type.html#defining-the-form-type. I used this link in the Forms component so will create an `xref` to it